frontend-ui/src/stores/tasks.ts

```diff
@@ -62,11 +63,29 @@ export const useTasksStore = defineStore('tasks', () => {
     }
   }
 
+  const fetchTask = async (id: string, hideSecrets: boolean = true) => {
+    const service = await authStore.getApiService('tasks')
+    try {
+      const response = await service.get<null, Task>(`/${id}`, { params: { hide_secrets: hideSecrets } })
```

Comment:
Passing `hide_secrets: False` and letting backend decide is probably easier / more accurate, it will save us from some nasty bugs where frontend and backend do not agree about wether to hide secrets or not.